jujugui/static/gui/src/app/components/env-switcher/test-env-switcher.js

```diff
@@ -206,10 +211,11 @@ describe('EnvSwitcher', function() {
     assert.equal(newEnv.args[0][0], 'admin');
     assert.equal(newEnv.args[0][1], 'new-env-1');
     assert.equal(newEnv.args[0][2], 'admin/gui');
-    assert.closeTo(newEnv.args[0][3].length, 31, 2);
+    assert.equal(newEnv.args[0][3], 'admin/gui');
```

Comment:
admin/gui isn't a hardcoded value anymore--it's just testing that the state-server name is the name that is then used for the template. Since we set up this "environment" with a state server name admin/gui, it uses the template also named admin/gui.